File: Data Structures & Algorithms/string-encode-and-decode/submission-0.py
class Solution:

    def encode(self, strs: List[str]) -> str:
        result = ''
        for val in strs:
            result += str(len(val)) + 't' + val
        return result



    def decode(self, s: str) -> List[str]:
        result = []
        print(s)
        while len(s) > 0:
            print(s)
            delimeterIndex = s.index('t')
            charCount = int(s[:delimeterIndex])
            end = delimeterIndex + charCount + 1
            result.append(s[delimeterIndex+1:end])
            s = s[end:]
        return result

File: Data Structures & Algorithms/daily-temperatures/submission-0.java
class Solution {
    public int[] dailyTemperatures(int[] temperatures) {
        int len = temperatures.length;
        int[] result = new int[len];
        for(int i = 0 ; i < len; i++) {
            for( int j = i; j < len; j++) {
                if(temperatures[i] < temperatures[j]) {
                    result[i] = j - i;
                    break;
                }
            }
        }
        return result;

    }
}
